policies/modules/ROOT/pages/policies-automated-overview.adoc

```diff
@@ -2,19 +2,19 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:page-aliases: api-manager::automated-policies-landing-page.adoc
+// :page-aliases: api-manager::automated-policies-landing-page.adoc
 
-Configuring and using automated policies enables your Anypoint Platform instance to comply with common security and logging requirements by automatically applying the same set of policies to all APIs running in a single environment. With policy automation, API owners can quickly design, build, and deploy secure and consistent APIs.
+Configuring and using automated policies enables your API instance to comply with common security and logging requirements by automatically applying the same set of policies to all APIs running in Flex Gateway, or in Mule. With policy automation, API owners can quickly design, build, and deploy secure and consistent APIs.
```

Comment:
This is a long drink of water. Any way to tighten?
Use automated policies to enforce security and logging requirements by applying the same policies to all APIs running in Flex Gateway or Mule.

Reply:
The last sentence of line 7 has some repetition with lines 9 and 11.

Consider "you" instead of "API owners".

Reply:
All fixed. Redundancies removed.